__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../app");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const endpoints= require('../endpoints.json')
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => connection.end());
+
+describe("/api", () => {
+  test("GET - status:200 - returns JSON with available endpoints ", () => {
```

Comment:
This test is fine, but bear in mind that as the JSON file updates, this test will still pass, so it's important to come and check it still works every time, even though the test will remain the same :)